src/ui/wxWidgets/wxUtilities.cpp

```diff
@@ -263,6 +263,33 @@ bool IsCurrentDesktopKde()
 #endif
 }
 
+bool wxUtilities::IsDisplayManagerX11()
+{
+#ifdef __WINDOWS__
+  return false;
+#elif defined __WXOSX__
+  return true;
```

Comment:
Since the function returns true not only for X11 maybe it needs a different name? How about `IsVirtualKeyboardSupported`?

Reply:
Good idea. The name is more generic and would be more appropriate if the check were different for different platforms. The routine was created mainly from a Linux perspective. :)